env/env_test.cc

```diff
@@ -166,6 +166,40 @@ TEST_F(EnvPosixTest, AreFilesSame) {
 }
 #endif
 
+#ifdef OS_LINUX
+TEST_F(EnvPosixTest, FilePermission) {
+  // Only works for Linux environment
+  if (env_ == Env::Default()) {
```

Comment:
do we need this conditional?

Reply:
maybe explaining in what case it evaluates to false (and why the test case can't work in those situations) would help justify it better.

Reply:
note this case is `TEST_F`, not `TEST_P`, so in this case the Env under test is not parameterized. So this will always evaluate to either false or true -- I am not sure which, but hope true otherwise the test will be a no-op.

Reply:
@xiaofeidu008 Thanks for checking :).